yarn.lock

```diff
@@ -1000,6 +1000,20 @@
     unique-filename "^1.1.1"
     which "^1.3.1"
 
+"@firebase/firestore@1.12.1":
```

Comment:
Should `@firebase/firestore` be here?

Reply:
Good question. I don't know why it shows up here.
Yarn shows It's pulled in by `workspace-aggregator-ce563d83-8dd9-4c27-b9d6-f5e2a2c4a6bd`. Looks like a generated thing, but I couldn't figure out where it is coming from. I cleaned workspace and `yarn`, still the same thing. 

For now I manually removed firestore from yarn.lock.